noxfile.py

```diff
@@ -60,7 +60,7 @@
 UNIT_TEST_EXTERNAL_DEPENDENCIES: List[str] = []
 UNIT_TEST_LOCAL_DEPENDENCIES: List[str] = []
 UNIT_TEST_DEPENDENCIES: List[str] = []
-UNIT_TEST_EXTRAS: List[str] = []
+UNIT_TEST_EXTRAS: List[str] = ["polars"]
 UNIT_TEST_EXTRAS_BY_PYTHON: Dict[str, List[str]] = {}
```

Comment:
Let's make sure we run some unit tests without polars to prevent accidental hard dependencies on it.

```suggestion
UNIT_TEST_EXTRAS: List[str] = []
UNIT_TEST_EXTRAS_BY_PYTHON: Dict[str, List[str]] = {"3.12": ["polars"]}
```